src/Aspire.Hosting.SqlServer/SqlServerBuilderExtensions.cs

```diff
@@ -72,12 +110,33 @@ public static IResourceBuilder<SqlServerDatabaseResource> AddDatabase(this IReso
         ArgumentNullException.ThrowIfNull(builder);
         ArgumentException.ThrowIfNullOrEmpty(name);
 
+        Dictionary<string, string> databaseConnectionStrings = [];
```

Comment:
Why do you need this? This call has all of the details for a single database right? You can just capture the  databaseConnectionString